packages/specification/schemas/components/MedicationRequest/CancelRequestMedicationRequest.yaml

```diff
@@ -155,9 +155,12 @@ properties:
           The duration of each issue within an eRD prescription. This must be expressed in days.
         properties:
           value:
-            type: integer
-            description: The number of days each issue in an eRD prescription is valid for.
-            example: 2
+            type: number
```

Comment:
This should not change. The expectedSupplyDuration should just be an integer as you can't have something valid for 28.5 days.